### ISSUE TITLE
Quest - Provide error handling for missing properties for configs  - Application crashes if appId is null

### ISSUE THREAD

**Issue by f-odhiambo** [opensrp/fhircore#955]
**Describe the bug**
Application crashed if appId is null or not defined or specifically property appId has not been initialized

**To Reproduce**
Steps to reproduce the behaviour:

1. Go to the Quest app
2. Click on the last Questionnaire 
3. See error

**Expected behaviour**
Add error handling in case the questionnaire has null properties required for rendering/filtering 

**Screenshots**
N/A

**Smartphone (please complete the following information):**
 - Device: Emulator and potentially physical app 
 - Android version: 12
 - Version 32

**Additional context**
Log
```
2022-01-17 16:35:58.690 8377-8377/org.smartregister.fhircore.quest E/AndroidRuntime: FATAL EXCEPTION: main
    Process: org.smartregister.fhircore.quest, PID: 8377
    kotlin.UninitializedPropertyAccessException: lateinit property appId has not been initialized
        at org.smartregister.fhircore.engine.configuration.ConfigurationRegistry.getAppId(ConfigurationRegistry.kt:51)
        at org.smartregister.fhircore.quest.data.patient.PatientRepository$fetchTestForms$2.invokeSuspend(PatientRepository.kt:148)
        at kotlin.coroutines.jvm.internal.BaseContinuationImpl.resumeWith(ContinuationImpl.kt:33)
        at kotlinx.coroutines.DispatchedTask.run(DispatchedTask.kt:106)
        at kotlinx.coroutines.scheduling.CoroutineScheduler.runSafely(CoroutineScheduler.kt:571)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.executeTask(CoroutineScheduler.kt:750)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.runWorker(CoroutineScheduler.kt:678)
        at kotlinx.coroutines.scheduling.CoroutineScheduler$Worker.run(CoroutineScheduler.kt:665)

```

**Comment by f-odhiambo** [comment]
@maimoonak Do we have a PR linked to this issue ?

**Comment by maimoonak** [comment]
@f-odhiambo I had tried multiple things to produce this error but seems like the appId is never null. The screen before login prevents any app being opened without a registered app. Also even if remember-me is off it still does have a valid appid in any case. The error is also seen in logs after app is trying to recover from a fatal crash and is preceded by two more exceptions, one is DeadObjectException and the real cause. I am assuming this might have been occurred due to another issue which has been fixed.
